### PR TITLE
I1581 dummy canvas report 3

### PR DESCRIPTION
**Pull request by Serhiy91**
- Added OperationalElementsReportCard

### CODE REVIEW

File: app/imports/client/react/canvas/components/CanvasReport/CanvasReportPage.js [hunk 1]
```diff
@@ -478,9 +615,7 @@ const CanvasReportPage = () => (
     </ReportCard>
     <CustomerInsightsReportCard customerSegments={data.customerSegments} />
     <ValueComponentsReportCard valuePropositions={data.valuePropositions} />
-    <ReportCard>
-      <ReportCardHeader>Operational elements</ReportCardHeader>
-    </ReportCard>
+    <OperationalElementsReportCard {...data} />
```
Comment (Dartv) [inline app/imports/client/react/canvas/components/CanvasReport/CanvasReportPage.js]: pass the exact data you need

File: app/imports/client/react/canvas/components/CanvasReport/OperationalElementsReportCard.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+const OperationalElementsReportCard = ({ customerSegments, keyPartners, keyActivities }) => (
+  <ReportCard>
+    <ReportCardHeader>
+      Operational elements
+      <Pull right>
+        <Icon size="2" name="industry" />
+      </Pull>
+    </ReportCardHeader>
+    <ReportOperationElementList title="Customer segments" items={customerSegments} />
```
Comment (Dartv) [inline app/imports/client/react/canvas/components/CanvasReport/OperationalElementsReportCard.js]: File name and component name is different. You should always name component exactly as file name

File: app/imports/client/react/canvas/components/CanvasReport/ReportOperationalElementList.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+const Wrapper = styled.div`
+  display: flex;
+  & > div {
+    width: 100%;
+    padding-right: 10px;
+  }
+`;
+
+const isOperationalElements = compose(
```
Comment (Dartv) [inline app/imports/client/react/canvas/components/CanvasReport/ReportOperationalElementList.js]: This name is confusing. I first thought it decides if documents are of operational element type. It's better to name it something like `hasOperationalElements`. The code could also be improved.
```js
compose(
  any(complement(isEmpty)),
  values,
);
```

File: app/imports/client/react/canvas/components/CanvasReport/ReportOperationalElementList.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+  complement(isEmpty),
+  concatAll,
+  values,
+);
+
+const ReportOperationalElementList = ({ title, items }) => {
+  const concatedDocs = concatLinkedDocs(items);
```
Comment (Dartv) [inline app/imports/client/react/canvas/components/CanvasReport/ReportOperationalElementList.js]: `concat` means it should return an array, but it returns Object. It's a bit confusing. I think a better name would be something like `buildLinkedDocsData`

File: app/imports/client/react/canvas/components/CanvasReport/ReportOperationalElementList.js [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+    standards = [],
+    risks = [],
+    nonconformities = [],
+    potentialGains = [],
+  } = concatedDocs;
+  const nonconformitiesAndGains = concat(nonconformities, potentialGains);
+  return isOperationalElements(concatedDocs) && (
+    <Fragment>
+      <hr />
+      <ReportHeader tag="h5">{title}</ReportHeader>
```
Comment (Dartv) [inline app/imports/client/react/canvas/components/CanvasReport/ReportOperationalElementList.js]: `label` instead?

**Comment by Serhiy91** [comment]
fixed https://github.com/Pliohub/Plio/pull/1607

**Pull request merged**